app/views/standard/additionalMetadataView.scala.html

```diff
@@ -3,6 +3,7 @@
 
 @import graphql.codegen.GetConsignmentFilesMetadata.getConsignmentFilesMetadata.GetConsignment.Files.FileMetadata
 
+@import views.html.partials.backLink
```

Comment:
Can add this to the existing import above

`@import views.html.partials.{transferReference, metadata, backLink}`